ferrocene/doc/internal-procedures/src/working-with-the-ci.rst

```diff
@@ -118,16 +90,24 @@ Set up the virtual environment:
 
 .. code-block:: bash
 
+   uv python install 3.12
    uv venv
-   source .venv/bin/activate
    uv pip sync requirements.txt
 
-You need to source the virtual enviroment each time you wish to use it:
+You can then source the virtual enviroment each time you wish to use it:
 
 .. code-block:: bash
 
    source .venv/bin/activate
 
+If you infrequently use Python scripts, or want to use them in a oneshot command:
+
+.. code-block:: bash
+
+   uv run $COMMAND
+
+For example, to run our licensing checks you can run ``reuse --include-submodules lint``
```

Comment:
Nit: I'd show this as a code-block with the `uv run` prefix.

Reply:
😅 That is a good idea and I thought I did that.